Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationEndpointProtectionPolicyWindows10/settings.json

```diff
@@ -0,0 +1,39 @@
+{
+    "resourceName": "IntuneDeviceConfigurationEndpointProtectionPolicyWindows10",
+    "description": "This resource configures an Intune Device Configuration Endpoint Protection Policy for Windows10.",
+    "permissions":    {
+    "graph":  {
+                  "delegated":  {
+                                    "read":  [
+                                                 {
+                                                     "name":  "DeviceManagementConfiguration.Read.All"
+                                                 },
+                                                 {
+                                                     "name":  "DeviceManagementConfiguration.ReadWrite.All"
```

Comment:
One more comment: Is this ReadWrite permission really necessary for reading the config?

Reply:
fixed